server/world/world.go

```diff
@@ -234,33 +234,69 @@ func (w *World) highestObstructingBlock(x, z int) int {
 	return w.ra[0]
 }
 
+// Opts ...
+type Opts struct {
```

Comment:
Would maybe rename this to `SetOpts` in case we ever add `Opts` of a different kind.